src/pages/events/index.page.tsx

```diff
@@ -2,8 +2,22 @@ import React from 'react';
 import DisplayRC from '../../components/EventsDisplay';
 import logo from '../../../public/reactdevske.svg';
 import Head from 'next/head';
+import type { GetStaticProps } from 'next';
 
-export const getStaticProps = async () => {
+type EventData = {
```

Comment:
This interface has been defined in `src/components/EventsDisplay/DisplayRC.tsx`. Perhaps we can create a shared interface and import to all the files using it

Reply:
@Lekipising Nice observation. Fixed in https://github.com/reactdeveloperske/reactdevske-website/pull/74/commits/760edb9682f1eed758952861e243f92906d6fe6d

Reply:
Awesome @antosan :ok_hand: